chore:Updated ujust dns wording

ShadowSlayer1441 opened the pull request:
Just fixed the working of line 26 to clearly show you need to select it to use vpn dns resolver

ShadowSlayer1441 committed:
Update dns.just

Just fixed the working of line 26 to clearly show you need to select it to use vpn dns resolver

RKNF404 commented on files/justfiles/dns.just:
```diff
@@ -23,7 +23,7 @@ dns-selector:
     echo "All resolvers support DNS-over-TLS (DoT) or DNS-over-QUIC (DoQ), DNS-over-HTTPS (DoH), and DNSSEC."
     echo "Please select which DNS resolver you would like to set:"
-    echo "    0) Network Resolver - security may vary (system default state)"
+    echo "    0) Network Resolver - security may vary (system default state) (recommended for VPN use with custom dns)"
```
> recommended for VPN use with custom dns

I think this needs a little clarification as to what this means. Perhaps referring specifically to DNS leak protection? Since most VPNs have documentation on this if they offer it.

ShadowSlayer1441 replied:
`(recommended when using VPN DNS leak protection)`
Should I call it required?
`(required when using VPN DNS leak protection)`

RKNF404 replied:
its not required, its just *much* better

ShadowSlayer1441 committed:
Update dns.just

changed to '(recommended when using VPN DNS leak protection)'

ShadowSlayer1441 committed:
Update dns.just

(select if using a VPN, this ensures VPN provided DNS is used to prevent DNS leaks)

ShadowSlayer1441 committed:
Update dns.just

(select if using a VPN, this ensures the VPN provided DNS is used to prevent DNS leaks)

ShadowSlayer1441 committed:
retrigger checks

ShadowSlayer1441 committed:
Merge branch 'live' into live

ShadowSlayer1441 committed:
[chore]:updating ujust dns script wording

ShadowSlayer1441 committed:
qMerge branch 'live' of github.com:ShadowSlayer1441/secureblue into live

ShadowSlayer1441 committed:
[chore]:updating ujust dns script wording

ShadowSlayer1441 committed:
[CHORE]:updating ujust dns script wording

ShadowSlayer1441 committed:
chore:updating ujust dns script wording

ShadowSlayer1441 committed:
chore:updating ujust dns script wording

ShadowSlayer1441 committed:
added my pub key

ShadowSlayer1441 committed:
Update build.yml optimize action run time
